app/managers.py

```diff
@@ -0,0 +1,39 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self._con = sqlite3.connect("cinema.db")
```

Comment:
i think, that, you could better named variable _con, more readable

in everything else, nice code

Reply:
thanks.